code/modules/clothing/head/f13factionhead.dm

```diff
@@ -497,6 +497,18 @@
 	icon_state = "legion-heavy"
 	item_state = "legion-heavy"
 	armor = list("melee" = 60, "bullet" = 45, "laser" = 60, "energy" = 15, "bomb" = 45, "bio" = 60, "rad" = 30, "fire" = 60, "acid" = 20, "wound" = 50)
+	
+/obj/item/clothing/head/helmet/f13/combat/legion
```

Comment:
These already exist, don't add something already in the code.

Reply:
They don't exist, I literally checked through f13factionhead.

Reply:
![image](https://user-images.githubusercontent.com/79063506/173596239-91ec624f-832a-4d73-b547-61ca23c7f297.png)

Reply:
Bottom of f13factionhead.dm

Reply:
removed em 👍